### PR TITLE
Upgrade to PHP 8.1

### PR DESCRIPTION
**Pull request by vehikl-galang**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -7,9 +7,9 @@
     ],
     "license": "MIT",
     "require": {
-        "php": ">=7.1.0",
+        "php": ">=8.1",
```
Comment (dennislindsey) [inline composer.json:10]: Any reason to drop support for php 7?

**Review by dennislindsey — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (vehikl-galang) [reply composer.json:10]: The release of `pragmarx/recovery` that this package is currently using doesn't support PHP 8.1 and can't be made backwards compatible.  Only LE is using this package and our goal is to move it to at least PHP 8.1 so this shouldn't cause any conflicts.

Comment (dennislindsey) [reply composer.json:10]: Can you confirm the composer require definition in LE for this package for me? Just want to make sure it's using a tagged release and not a branch name.

Comment (vehikl-galang) [reply composer.json:10]: It's using `1.0.3-rc3`, so we can safely merge this in and make a new release for the PHP upgrade.

**Pull request merged**